redundant-bmc/src/manager.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <sdbusplus/async.hpp>
+#include <xyz/openbmc_project/State/BMC/Redundancy/server.hpp>
+
+namespace rbmc
+{
+
+using RedundancyIntf =
+    sdbusplus::xyz::openbmc_project::State::BMC::server::Redundancy;
+using RedundancyInterface = sdbusplus::server::object_t<RedundancyIntf>;
+
+/**
+ * @class Manager
+ *
+ * Manages the high level operations of the redundant
+ * BMC functionality.
+ */
+class Manager
+{
+  public:
+    Manager() = delete;
+    ~Manager() = default;
+    Manager(const Manager&) = delete;
+    Manager& operator=(const Manager&) = delete;
+    Manager(Manager&&) = delete;
+    Manager& operator=(Manager&&) = delete;
+
+    /**
+     * @brief Constructor
+     *
+     * @param ctx - The async context object
```

Comment:
Param is in/out?

Reply:
I've  seen in other repos where because out params are so rare that there's no need to do [in], and then [out] or [inout] can be specified when necessary, so I've gone that direction here as well.

Reply:
ok :(

Reply:
eh,  I changed it. and updated my vim abbreviation.